### PR TITLE
WIP:coreboot blogpost

### PR DESCRIPTION
**Pull request by KBanecka**

### CODE REVIEW

**Comment by artur-rs** [comment]
@KBanecka is it still WIP? Missing abstract, cover etc.

File: blog/content/post/2020-07-27-coreboot_blogpost.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+step of 3mdeb Firmware Training path. It provides solid grounds for further
+development towards proficiency in coreboot firmware platform enginering.
+
+![graf](/blog/static/img/training_path.png)
+
+#### corebootability
+
+Most modern platforms require closed-source BIOS for booting purposes. However,
+BIOS is not featured by flexibility and easiness in supporting custom hardware
+or maintenance. coreboot, formerly known as LinuxBIOS, may use a real operating
```
Comment (miczyg1) [inline blog/content/post/2020-07-27-coreboot_blogpost.md:47]: This is not right:
```
use a real operating system to boot operating system
```

File: blog/content/post/2020-07-27-coreboot_blogpost.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+by hardware vendors.
+
+> Do as much as needed, then jump straight payload.
+>
+> – coreboot.
+
+
+#### Why worthy
+
+The training will give you an insight
```
Comment (miczyg1) [inline blog/content/post/2020-07-27-coreboot_blogpost.md:62]: The line ends too early

File: blog/content/post/2020-07-27-coreboot_blogpost.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+* boot flow from power on to system take off
+* coreboot building environment
+* coreboot's customized built
+* SPI flash theory of operation
+* flashing tools and methods
+* coreboot's project structure
+* move around in project - apply changes
+* hands-on workshop using previously gained knowledge
+
+#### I am an embedded engineer. What is your superpower?
```
Comment (miczyg1) [inline blog/content/post/2020-07-27-coreboot_blogpost.md:96]: need new line after the header

File: blog/content/post/2020-07-27-coreboot_blogpost.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+we've responded to this need by providing reliable and professional training
+that covers essential basics. We are proud of the way we've managed to hammer
+out years of experience into modular, easy-to-follow 4-hours training. **In this
+new blogpost series we will be publishing coreboot fundamentals mini-course**
+equipped with coreboot essentials. [coreboot Fundamentals
+Training](https://training.3mdeb.com/courses/coreboot-fundamentals) is the first
+step of 3mdeb Firmware Training path. It provides solid grounds for further
+development towards proficiency in coreboot firmware platform enginering.
+
+![graf](/blog/static/img/training_path.png)
```
Comment (miczyg1) [inline blog/content/post/2020-07-27-coreboot_blogpost.md:41]: The paths should be just `img/...`

As it is now, the images are not rendering. Fix all the occurrences please

File: blog/content/post/2020-07-27-coreboot_blogpost.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+knowledge gets more and more value over time what provides stable work position
+with above-average salary and greater recognition inside the industry. Embedded engineer is a
+kind of a hybrid that brings together electrical engineering and logic skills
+of a software programming. His path brings lots of opportunities both
+in commercial and open ecosystem and even greater satisfacion thanks to the role
+of mediator between hardware and software worlds. Embedded engineers can be
+proudly called the alchemists of computing industry.
+
+If you think we can help in improving the security of your firmware or you are
+looking for a training that can boost your team of developers inside UEFI, coreboot or yocto area, feel free to [book a call with us](https://calendly.com/3mdeb/consulting-remote-meeting)
```
Comment (miczyg1) [inline blog/content/post/2020-07-27-coreboot_blogpost.md:109]: Yocto is written in captial letter. If this is a bug in the template please fix it.

File: blog/content/post/2020-07-27-coreboot_blogpost.md (hunk 1 above)
Comment (KBanecka) [reply blog/content/post/2020-07-27-coreboot_blogpost.md]: done

File: blog/content/post/2020-07-27-coreboot_blogpost.md (hunk 2 above)
Comment (KBanecka) [reply blog/content/post/2020-07-27-coreboot_blogpost.md]: done

File: blog/content/post/2020-07-27-coreboot_blogpost.md (hunk 3 above)
Comment (KBanecka) [reply blog/content/post/2020-07-27-coreboot_blogpost.md:94]: done

File: blog/content/post/2020-07-27-coreboot_blogpost.md (hunk 4 above)
Comment (KBanecka) [reply blog/content/post/2020-07-27-coreboot_blogpost.md]: done

File: blog/content/post/2020-07-27-coreboot_blogpost.md (hunk 5 above)
Comment (KBanecka) [reply blog/content/post/2020-07-27-coreboot_blogpost.md]: done

**Pull request merged**